.github/workflows/e2e.yml

```diff
@@ -0,0 +1,39 @@
+name: E2E Tests
+
+on:
+  pull_request:
+    branches: [trunk]
+
+jobs:
+  test:
+    name: Playwright e2e Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v4
+        with:
+          node-version: "20.15.1"
```

Comment:
In the long run, it would be easier to use `.nvmrc` as the source of true:

https://github.com/WordPress/secure-custom-fields/blob/157ea08aeabf970a60451d1ebcd322409e779020/.nvmrc#L1

See here:

https://github.com/WordPress/gutenberg/blob/bdb16796eb85051b9d5d7b00fcb773973642a5e7/.github/workflows/static-checks.yml#L29-L32

Reply:
Handling via #93